packages/api/src/platforms/vtex/resolvers/order.ts

```diff
@@ -0,0 +1,15 @@
+import type { Resolver } from '..'
+import type { PromiseType } from '../../../typings'
+import type { Query } from './query'
+
+type QueryUserOrder = PromiseType<ReturnType<typeof Query.userOrder>>
+
+export type Root = QueryUserOrder
+
+export const UserOrder: Record<string, Resolver<Root>> = {
+  orderId: ({ orderId }) => orderId,
+  totals: ({ totals }) => totals,
+  items: ({ items }) => items,
+  shippingData: ({ shippingData }) => shippingData,
+  paymentData: ({ paymentData }) => paymentData,
+}
```

Comment:
Here we can map the frontend <> OMS API data that we need to move some processing from the frontend to the backend.

Reply:
This file is just an example that we can use to map some frontend data. But actually, all the data from the API is available to the frontend [here the logisticsInfo](https://github.com/vtex/faststore/blob/feabceebd09f0f9303f143ac925158437e964e02/packages/core/src/pages/account/orders/%5Bid%5D.tsx#L294). Maybe after we can filter to just the fields that we are using.

Reply:
actually, this file was superseded by the other resolver orderResult. I'll remove it.